docs/ops/python_shell.md

```diff
@@ -24,6 +24,21 @@ under the License.
 
 Flink comes with an integrated interactive Python Shell.
 It can be used in a local setup as well as in a cluster setup.
+See the [local setup page](../getting-started/tutorials/local_setup.html) for more information about how to setup a local Flink.
+You can also [build a local setup from source](../flinkDev/building.html).
+
+<span class="label label-info">Note</span> The Python Shell will run the command “python”. Please run following command to confirm that the command “python” in current environment points to Python 3.5+:
+
+{% highlight bash %}
+$ python --version
+# the version printed here must be 3.5+
+{% endhighlight %}
+
+<span class="label label-info">Note</span> Using Python UDF in Python Shell requires apache-beam 2.15.0. Run following command to confirm that it meets the requirements before run the Shell in local mode:
```

Comment:
Run **the** following command to confirm that it meets the requirements before **running** the Shell in local mode

Same for other places.